cmd/wallet-web/src/pages/Signup.vue

```diff
@@ -227,6 +251,9 @@ export default {
     handleSuccess() {
       this.$router.push(this.redirect);
     },
+    checkIfLoggedIn() {
+      window.location.href = this.redirect;
```

Comment:
you are checking or redirecting?

Reply:
updated function name, redirecting

Reply:
add a TODO for now, it should be router.push(), we shouldn't reload screen